src/main/resources/eu/markov/jenkins/plugin/mvnmeta/MavenMetadataParameterDefinition/index.jelly

```diff
@@ -54,6 +54,10 @@
           </select>
         </j:otherwise>
       </j:choose>
+      <j:set var="currentArtifactInfo" value="${it.currentArtifactInfo}"/>
```

Comment:
You're right! I introduced the ``currentArtifactInfo`` var because ``it.currentArtifactInfo`` is used in more than just one place. Just missed to replace the second occurrence. I'll fix that.